src/lib/CurlHelper.js

```diff
@@ -78,7 +79,8 @@ export class CurlHelper {
     let url = this.getUrl();
 
     if (this.getQueryString() != "") {
-      url = (url.charAt(url.length - 1) == '/')  ? url.substr(0, url.length - 1) : url; 
+      url =
+        url.charAt(url.length - 1) == "/" ? url.substr(0, url.length - 1) : url;
```

Comment:
Please keep this variable assignement on the same line. The short-hand if statement could possibly be seperated on multiple lines however.

Reply:
I think we're on the limit for this line ;) . My Prettier made that change, but I think you're absolutely right in this case.